en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the `silverstripe` GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+The purpose of this policy is to provide clear guidance on the types of versions and version releases, including their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+## Verion numbering
+
+### Semantic versioning
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its version numbering of individual modules such as `silverstripe/framework`
+
+For instance for the version string `6.1.5`:
+
+- the *major version* is the first number in the version string - `6`
+- the *minor version* is the second number in the version string - `1`
+- the *patch version* is the third number in the version string - `5`
+
+### CMS versions
+
+CMS versions are what are commonly referenced to when people refer to the version of the CMS. They do not include a patch version as it applies to the CMS as whole, which is a collection of multiple modules.
+
+- *CMS major versions* only include the *major version* number from the version string. For instance "CMS 6" has a *major version* of "6" and any *minor version* or *patch* version.
+
+- *CMS minor versions* include the *major version* and *minor version* from the version string. For instance "CMS 6.1" has a *major version* of "6" and a *minor version* of "1".
+
+> [!IMPORTANT]
+> Silverstripe CMS support timelines are structured exclusively around *CMS minor versions*
+
+## Definition of version releases
+
+Silverstripe CMS aims to deliver regular releases at predefined intervals with a clear support timeline. This is to allow Silverstripe CMS project owners to plan upgrades ahead of time.
+
+Because of the use of semantic versioning for version numbering, there are three version release types which allow for a different levels of changes:
+
+- [Major version release](#major-version-release)
+- [Minor version release](#minor-version-release)
+- [Patch version release](#patch-version-release)
+
+### Major version (not a release)
+
+When the first minor version release (defined below) of a major version is performed, it gets the status of being both a major release, and a minor release. For instance the "CMS 6.0" minor release is also called the "CMS 6 major release"
```

Comment:
This bit is needs to change, however I'm not sure how to write this

We have probably too many concepts floating around:

- Semantic versions
- CMS versions
- CMS version releases
- Major releases ... which aren't actually a thing (only minor releases are things)